neutopia/src/chest.rs

```diff
@@ -21,6 +21,57 @@ impl Chest {
 
         Ok(())
     }
+
+    pub fn get_item_name(&self) -> String {
+        return match self.item_id {
+            0 => format!("{} bombs", self.arg),
+            1 => format!("Medicine"),
+            2 => format!("Firewand"),
+            3 => format!("Sky Bell"),
+            4 => format!("Wings"),
+            5 => format!("Moonbeam Moss level {}", self.arg),
+            6 => format!("Magic Ring"),
+            7 => format!("Placeholder"),
+            8 => match self.arg {
+                1 => format!("Starter Sword"),
+                2 => format!("Blue Sword"),
```

Comment:
I think the three upgrades for sword, shield, and armor, are named:
- Bronze,
- Iron
- Strongest